src/NinjaTools.FluentMockServer/Xunit/Attributes/IsolatedMockContextAttribute.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using NinjaTools.FluentMockServer.Models;
+using Xunit.Sdk;
+
+namespace NinjaTools.FluentMockServer.Xunit.Attributes
+{
+    /// <summary>
+    /// Unit tests marked with this <see cref="Attribute"/> will create <see cref="Expectation"/> inside an isolated
+    /// Context.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public class IsolatedMockServerSetupAttribute : BeforeAfterTestAttribute
+    {
+    }
+
+
+    namespace NinjaTools.FluentMockServer.Xunit.Attributes
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty namespace.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4927362)